src/resources/views/partials/channels_modal.blade.php

```diff
@@ -56,7 +56,7 @@
       </div>
       {{-- /.modal-body --}}
       <div class="modal-footer">
-        <a href="{{ route('seatnotifications.notification.unsubscribe.channel') }}" type="button" class="btn btn-danger pull-left">Unsubscribe</a>
+        <button type="button" data-dismiss="modal" class="btn btn-default pull-left">Close</button>
```

Comment:
This is fine for the moment until we have resolved the `driver_id` discussion. The modal in version 2.0 would basically work differently then in version v 1.1.2: i used the model to subscribe AND update a subscription. 

I would actually to see this again but i am curious what your opinion is.